<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MouseConfig</class>
 <widget class="QWidget" name="MouseConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_43">
     <property name="text">
      <string>&lt;b&gt;Mouse&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Motion</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="1" column="3">
       <widget class="QLabel" name="label_35">
        <property name="text">
         <string>High</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_36">
        <property name="text">
         <string>Fast</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_37">
        <property name="text">
         <string>Sensitivity:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_38">
        <property name="text">
         <string>Low</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSlider" name="mouseThreshold">
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>110</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_39">
        <property name="text">
         <string>Acceleration:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSlider" name="mouseAccel">
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>110</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_40">
        <property name="text">
         <string>Slow</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_41">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_42">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Double click interval:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="doubleClickInterval">
     <property name="suffix">
      <string> ms</string>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Wheel scroll lines:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="wheelScrollLines"/>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QCheckBox" name="mouseLeftHanded">
     <property name="text">
      <string>Left handed (Swap left and right mouse buttons)</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QCheckBox" name="singleClick">
     <property name="text">
      <string>Single click to activate items</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>mouseAccel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_41</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mouseThreshold</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_42</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
